plugins/module_utils/scaleway.py

```diff
@@ -13,7 +13,7 @@
 
 def scaleway_argument_spec():
     return dict(
-        api_token=dict(required=True, fallback=(env_fallback, ['SCW_TOKEN', 'SCW_API_KEY', 'SCW_OAUTH_TOKEN', 'SCW_API_TOKEN']),
+        api_token=dict(required=True, fallback=(env_fallback, ['SCW_TOKEN', 'SCW_API_KEY', 'SCW_OAUTH_TOKEN', 'SCW_API_TOKEN', 'SCW_SECRET_KEY']),
```

Comment:
This would rather be a feature than a bugfix. It would be better to add this in another PR, since with a new feature in it, it cannot be backported to the stable-2 branch.